python-stdlib/logging/logging.py

```diff
@@ -8,68 +9,116 @@
 NOTSET = 0
 
 _level_dict = {
-    CRITICAL: "CRIT",
+    CRITICAL: "CRITICAL",
     ERROR: "ERROR",
-    WARNING: "WARN",
+    WARNING: "WARNING",
     INFO: "INFO",
     DEBUG: "DEBUG",
+    NOTSET: "NOTSET",
 }
 
-_stream = sys.stderr
+loggers = {}
+default_fmt = "%(levelname)s:%(name)s:%(message)s"
+default_datefmt = "%Y-%m-%d %H:%M:%S"
```

Comment:
I think it's best to prefix these with `_` to indicate they are private variables to this module (that's how `_loggers` was originally).